app/controllers/sealions.rb

```diff
@@ -0,0 +1,19 @@
+get "/sealions" do
+  File.read(File.join("public","sealions.html"))
+end
+
+get "/sealions/:id" do
+  File.read(File.join("public","sealion.html"))
+end
+
+post "/sealions" do
+  redirect "/sealions"
```

Comment:
Usually, you'd redirect to "/sealions/#{the_new_sealion_that_was_created.id}", but you don't have one of those yet. ;)